packages/palette/src/elements/Image/LazyImage.tsx

```diff
@@ -9,9 +9,10 @@ import {
 } from "styled-system"
 import { color } from "../../helpers/color"
 import { Box, BoxProps } from "../Box"
+import { withoutStyleProps } from "../WithoutStyleProps"
```

Comment:
Can the casing of this be consistent 1:1, either all upper-case or (preferrably) lower-case (since it's technically a HOC)?